CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### Features
+
+* Add Winston logger builder for other components to use
+
+### [8.2.1](https://github.com/awslabs/fhir-works-on-aws-interface/compare/v8.2.1...v8.1.1) (2021-04-06)
```

Comment:
```suggestion
### [8.2.1](https://github.com/awslabs/fhir-works-on-aws-interface/compare/v8.2.1...v8.1.1) (2021-04-06)

### Features

* Add Winston logger builder for other components to use

```

Reply:
Typically we make the release a separate PR

Reply:
+1. 
Revert all changes to changelog and make a separate PR similar to https://github.com/awslabs/fhir-works-on-aws-routing/pull/76 using https://www.npmjs.com/package/standard-version

When merging this PR, make sure that the commit msg indicates that this is a breaking change. e.g. https://github.com/awslabs/fhir-works-on-aws-interface/commit/cf497a0f565821effe6169df44202c02a00eef95

That will result a major version bump and the breaking change highlighted in the cheangelog e.g. https://github.com/awslabs/fhir-works-on-aws-interface/pull/51/files